documentation/API.md

```diff
@@ -372,18 +372,18 @@ The return object contains:
 
 The internal function called by this endpoint is the same one run by the ["generateDoc" action](List-of-Action-plugins.md).
 
-### Data Views
+#### Data Views
 
 GET:
 `/data-views`
-`/data-views/table/<tableName>`
-`/data-views/table/<tableName>/item/<id>`
+`/data-views/<dataViewCode>`
+`/data-views/<dataViewCode>/<itemId>`
```

Comment:
I also took the opportunity to simplify the url structure for these endpoints -- having the word `table` and `item` in there was unnecessarily verbose.

Reply:
You were thinking a bit Githubby for a moment